pycqed/measurement/openql_experiments/generate_QCC_cfg_ramiro.py

```diff
@@ -214,6 +214,22 @@ def generate_config(filename: str,
         },
 
         "gate_decomposition": {
```

Comment:
This file was created for a single experiment, very narrow in focus. It should now be deleted!